addons/calendar/static/src/views/attendee_calendar/attendee_calendar_model.js

```diff
@@ -77,6 +77,16 @@ export class AttendeeCalendarModel extends CalendarModel {
         );
         const currentPartnerId = this.user.partnerId;
         if (!isEveryoneFilterActive) {
```

Comment:
It does not seems to work when isEveryoneFilterActive is true.
Declined events are still dislayed even when hide refused events is checked. Is this normal ?

Reply:
When isEveryoneFilterActive is true then it'll hide event if everyone on the event refused that event.